packages/ibm-products/src/components/NotificationsPanel/NotificationsPanel.tsx

```diff
@@ -227,6 +233,11 @@ export interface NotificationsPanelProps {
    */
   todayLabel?: string;
 
+  /**
+   * Reference to trigger button
+   */
+  triggerButtonref?: RefObject<any>;
```

Comment:
Better change the prop name `triggerButtonref` to triggerButtonRef

Reply:
Thanks @makafsal.
Updated now!